packages/twenty-front/src/modules/object-record/record-table/record-table-header/components/RecordTableHeaderFirstCell.tsx

```diff
@@ -21,13 +21,11 @@ import { cx } from '@linaria/core';
 import { useState } from 'react';
 import { isDefined } from 'twenty-shared/utils';
 
-const StyledColumnHeadContainer = styled.div`
-  cursor: pointer;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-
-  overflow: hidden;
+const StyledPlusButtonWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 2;
```

Comment:
Can you please use TABLE_Z_INDEX.headerColumnswithoutGroups.headerColumnsSticky and check if it still works? Thanks!